packages/commonwealth/client/scripts/navigation/CommonDomainRoutes.tsx

```diff
@@ -48,7 +48,7 @@ const ChainEntityLinkRedirectPage = lazy(
 const SnapshotProposalLinkRedirectPage = lazy(
   () => import('views/pages/snapshot_proposal_link_redirect')
 );
-const FeedPage = lazy(() => import('views/pages/feed'));
+const FeedPage = lazy(() => import('views/pages/Feed'));
```

Comment:
May need to revert this (could not get this change to build on `experimental` unclear why